content/post/mirror-help/raspberry-pi-os-images.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "raspberry-pi-os-images"
+tags = ["mirror-help"]
+author = "danyang685"
++++
+
+主仓库地址： http://downloads.raspberrypi.org/
+
+**使用方法**
+
+进入以下包含 Raspbreey Pi OS 系统镜像的目录，选择`.zip`文件下载即可。推荐使用 [balenaEtcher](https://www.balena.io/etcher/) 将镜像写入 SD 卡
+
+- [raspios_lite_armhf](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_lite_armhf/images/)：精简版 Raspbreey Pi OS 系统（下载大小约2.3GB）
```

Comment:
Please use markdown table for this section

Reply:
Good idea!